data/schema.graphql

```diff
@@ -0,0 +1,6896 @@
+directive @principalField on FIELD
```

Comment:
Is this meant as the schema for tests to work or actual transformation of production code? In case of the latter, let’s not pull it into this repo and risk it going out of sync, but rather take it as a CLI argument when invoking the transformer.